ansible_collections/arista/avd/docs/plugins/Modules and action plugins/eos_validate_state_reports.md

```diff
@@ -0,0 +1,65 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# eos_validate_state_reports
```

Comment:
Consider changing this to `# EOS validate state reports`,in the navigation it looks a bit funky

Reply:
this is auto generated